catalogue/webapp/pages/works/[workId]/items.tsx

```diff
@@ -415,7 +424,7 @@ export const getServerSideProps: GetServerSideProps<
 
     return {
       props: serialiseProps({
-        transformedManifest: displayManifest,
+        transformedManifest: toCompressedTransformedManifest(displayManifest),
```

Comment:
Should we call this compressedTransformedManifest at this point?

Reply:
Yes, good suggestion, done.